rest_lights.cpp

```diff
@@ -2145,6 +2145,10 @@ int DeRestPluginPrivate::setWarningDeviceState(const ApiRequest &req, ApiRespons
             {
                 task.options = 0x12;
             }
+            else if (taskRef.lightNode->modelId() == QLatin1String("SIRZB-110"))    // Doesn't support strobe
+            {
+                task.options = 0x13;

```

Comment:
0x13 ?

I can tell this is different from 0x12. But it would be nice to know what 0x12/0x13 means at least a comment.

Reply:
Added comment